src/Mod/Tux/NavigationIndicatorGui.py

```diff
@@ -514,8 +522,10 @@ def onOrbitShow():
     gOrbit.blockSignals(True)
     if pView.GetInt("OrbitStyle", 1):
         aTrackball.setChecked(True)
-    else:
+    elif pView.GetInt("OrbitStyle", 0):
         aTurntable.setChecked(True)
+    else:
```

Comment:
I see an issue here because when using the menu, I don't get the orbit style I want (I cannot set Turntable). Why not set an if, elif, elif to catch all possible 3 orbit styles?

Reply:
Like this ?